main.go

```diff
@@ -75,6 +78,27 @@ Sample usage:
 }
 
 func main() {
+	ctx := context.Background()
+	lf, err := conversion.SetupLogFile()
+	if err != nil {
+		fmt.Printf("\nCan't set up log file: %v\n", err)
+		panic(fmt.Errorf("can't set up log file"))
+	}
+	defer conversion.Close(lf)
+
+	// TODO: Remove this check and always run HB in subcommands mode once
+	// global command line mode is deprecated. We can also enable support for
+	// top-level flags in subcommand then.
+	if os.Args[1] != "" && !strings.HasPrefix(os.Args[1], "-") {
```

Comment:
For `go run main.go schema ` we would need os.Args[3] right? Do we only plan on supporting it for built binaries?